### ISSUE TITLE
Add command to make a cursor, select the word below the cursor and goto next

### ISSUE THREAD

**Issue by pengux** [gabesoft/evil-mc#51]
In vim-easymotion, when pressing C-n, it will select the word below the cursor, pressing it again will find the next occurrence, selects it and create another cursor. It would be nice to have a command which replicate this behaviour. Below is a screenshot of what to expect when pressing C-n twice on the word `dotspacemacs`

![image](https://cloud.githubusercontent.com/assets/2286687/24323181/5fc99d36-1172-11e7-80da-3760e6057ddd.png)

**Comment by gabesoft** [comment]
Evil-mc works almost the same except you have to select the word first. Otherwise, it uses the whole symbol under cursor

**Comment by pengux** [comment]
But currently, the selections are lost when a cursor is made. The vim-easymotion behaviors saves keystrokes as most of the time you still want to have the selection left to either cut/replace. I'm happy to make a PR if you could tell me where in the code to look?

**Comment by gabesoft** [comment]
That's true it would save 2 keystrokes `vb` although it would cost 1 if the selection is not needed. Let me think about the best way to implement it. Also, we'll probably want to have an option to keep the old behavior if desired.

**Comment by pengux** [comment]
It saves another keystroke if it could select the word automatically instead of user has to make a selection beforehand. I'd suggest to create new commands for this so the old behavior is kept. Perhaps something like `evil-mc-select-make-and-goto-next-match`

**Comment by gabesoft** [comment]
Yeah a new command (and a new shortcut) is probably better

**Comment by dieggsy** [comment]
@gabesoft since you mentioned it here, it would be nice to make what is selected by default (symbol, word, etc.)
